test/wait/awaitility.go

```diff
@@ -63,9 +64,25 @@ func (a *ToolchainAwaitility) WaitForCheInstallationToDelete(name string) error
 	})
 }
 
+func (a *ToolchainAwaitility) WaitForTektonInstallationToDelete(name string) error {
```

Comment:
could be named `WaitForTektonInstallationToBeDeleted`, but that's a minor suggestion (same for `WaitForCheInstallationToDelete`)

Reply:
I noticed that too. Yeah I can change it

Reply:
done 94b03d9